springboot/src/main/java/pos/model/User.java

```diff
@@ -119,8 +134,8 @@ public boolean equals(Object obj) {
 
 /*são os acesso do usuário  	ROLE_ADMIN, ROLE_FUNCIONARIO...*/
 @Override
-public Collection<? extends GrantedAuthority> getAuthorities() {
-	
+public Collection getAuthorities() {
```

Comment:
The expression of type Collection needs unchecked conversion to conform to Collection<? extends GrantedAuthority>

Este warning é apresentado nas seguintes classes:
JWTTokenAuthenticationService
UserService

Teremos problemas ?